src/components/calcite-color-picker/calcite-color-picker.tsx

```diff
@@ -462,22 +506,53 @@ export class CalciteColorPicker {
     } = this;
     const selectedColorInHex = color ? color.hex() : null;
     const hexInputScale = scale !== "s" ? "m" : scale;
-    const { colorFieldAndSliderInteractive } = this;
+    const {
+      colorFieldAndSliderInteractive,
+      scopeTop,
+      scopeLeft,
+      hueScopeLeft,
+      hueScopeTop,
+      scopeOrientation
+    } = this;
     const elementDir = getElementDir(el);
     const noColor = color === null;
-
+    const vertical = scopeOrientation === "vertical";
     return (
       <div class={CSS.container}>
-        <canvas
-          class={{
-            [CSS.colorFieldAndSlider]: true,
-            [CSS.colorFieldAndSliderInteractive]: colorFieldAndSliderInteractive
-          }}
-          onMouseEnter={this.handleColorFieldAndSliderMouseEnterOrMove}
-          onMouseLeave={this.handleColorFieldAndSliderMouseLeave}
-          onMouseMove={this.handleColorFieldAndSliderMouseEnterOrMove}
-          ref={this.initColorFieldAndSlider}
-        />
+        <div class={CSS.colorFieldAndSliderWrap}>
+          <canvas
+            class={{
+              [CSS.colorFieldAndSlider]: true,
+              [CSS.colorFieldAndSliderInteractive]: colorFieldAndSliderInteractive
+            }}
+            onMouseEnter={this.handleColorFieldAndSliderMouseEnterOrMove}
+            onMouseLeave={this.handleColorFieldAndSliderMouseLeave}
+            onMouseMove={this.handleColorFieldAndSliderMouseEnterOrMove}
+            ref={this.initColorFieldAndSlider}
+          />
+          <div
+            aria-label={vertical ? this.intlValue : this.intlSaturation}
+            aria-valuemax="100"
+            aria-valuemin="0"
+            aria-valuenow={color?.hsv().array()[vertical ? 2 : 1] || "0"}
+            class={CSS.scope}
+            onKeyDown={this.handleScopeKeydown}
+            role="slider"
+            style={{ top: `${scopeTop}px`, left: `${scopeLeft}px` }}
+            tabindex="0"
+          />
+          <div
+            aria-label={this.intlHue}
+            aria-valuemax="360"
+            aria-valuemin="0"
+            aria-valuenow={color?.hue()?.toFixed(0) || "0"}
```

Comment:
For the no-color case, I think we need to remove `aria-valuenow` and possibly use `aria-valuetext` to provide a description:

From https://www.w3.org/TR/wai-aria/#aria-valuenow:

> If the current value is not known (for example, an indeterminate progress bar), the author SHOULD NOT set the aria-valuenow attribute. If the aria-valuenow attribute is absent, no information is implied about the current value.

> When the rendered value cannot be accurately represented as a number, authors SHOULD use the aria-valuetext attribute in conjunction with aria-valuenow to provide a user-friendly representation of the range's current value.

Reply:
I was yelled at by the a11y tools when I didn't provide a now value. But tbh, there is a color field, and it has a scope, even when it's empty. So to me, value now should reflect the position of the scope.

Reply:
This is not correct though. Having no hue is not the same as a hue of 0. We can pick this up in that follow-up a11y issue you proposed.

Reply:
Yeah, you could remove the fallback to `"0"`. Again, I have not been able to get that condition to ever happen, though.

Reply:
So you can't omit `now`. Our accessibility check inside the test suite fails you if it's omitted. Is there a way to tell the a11y check to ignore certain rules in certain checks?

Reply:
looks like the axe core folks recommend using "0" if it's unknown... 

https://github.com/dequelabs/axe-core/issues/2639#issuecomment-728210415

Reply:
Thanks for digging into this. It's really useful info. Seems like that and `aria-valuetext` is the winning combo. 💥 👊🏼 💥